code/modules/library/bibles.dm

```diff
@@ -81,8 +81,23 @@ GLOBAL_LIST_INIT(bibleitemstates, list(
 /obj/item/book/bible/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE_HOLY)
+	AddComponent(\
+		/datum/component/bullet_intercepting,\
+		active_slots = ITEM_SLOT_SUITSTORE,\
+		on_intercepted = CALLBACK(src, PROC_REF(on_intercepted_bullet)),\
+	)
 	carve_out()
 
+/// Destroy the bible when it's shot by a bullet
+/obj/item/book/bible/proc/on_intercepted_bullet(mob/living/victim, obj/projectile/bullet)
+	victim.add_mood_event("blessing", /datum/mood_event/blessing)
+	playsound(victim, 'sound/magic/magic_block_holy.ogg', 50, TRUE)
+	victim.visible_message(span_warning("\The [src] takes \the [bullet] in [victim]'s place!"))
+	var/obj/structure/fluff/paper/stack/pages = new(get_turf(victim))
+	pages.dir = pick(GLOB.alldirs)
+	victim.dropItemToGround(src, force = TRUE, silent = TRUE)
+	qdel(src)
```

Comment:
Maybe I'm wrong, but it should be safe to qdel the object without having to drop it on the ground.

Reply:
> Maybe I'm wrong, but it should be safe to qdel the object without having to drop it on the ground first.

When I didn't do this then it would leave their suit storage slot looking a little weird.
I can try doing the tearing thing instead though.